processing/api.py

```diff
@@ -60,13 +58,12 @@ def report_done(self, recording, newKey, newMimeType, metadata = {}):
         r = requests.put(self.url, data=params)
         r.raise_for_status()
 
-    def tag_recording(self, recording, label, confidence):
-        tag = {"automatic": True, "confidence": confidence}
+    def tag_recording(self, recording, label, metadata):
+        tag = metadata
```

Comment:
You might want to make this `tag = metadata.copy()` otherwise you're changing the input which is probably not expected.